test/suites/dev-tanssi/fees/test_fee_balance_transfer.ts

```diff
@@ -54,10 +56,12 @@ describeSuite({
                 const basePlusWeightFee = (
                     await polkadotJs.call.transactionPaymentApi.queryWeightToFee(info2.weight)
                 ).toBigInt();
-                expect(basePlusWeightFee).to.equal(1000000n + 1630678n);
+                // TODO: where do these values come from?
+                expect(basePlusWeightFee).to.equal(1000000n + 1479873n);
```

Comment:
1000000n is the base fee, which I believe is the fee paid for a transaction with 0 weight, and the other value comes from the weight of the extrinsic and it's only here to detect when it changes. In this case it's the weight of balances.transfer, which I guess has changed with the release.

Reply:
You can try moving the +1 here by changing the value to 1479874n.

Reply:
yeah but I want  to understand where it i coming from

Reply:
Oh, you can't move it here because then this expect would fail and you would need a -1 here...

Reply:
I think its coming from here: https://github.com/paritytech/polkadot-sdk/blob/2c7166eb9c82bedb02bba9882bc5aa5cd38b8fad/substrate/frame/transaction-payment/src/lib.rs#L761

Reply:
I don't think so, git blame shows 2 years ago. If it's in that file, then it must be this commit https://github.com/paritytech/polkadot-sdk/commit/2c7166eb9c82bedb02bba9882bc5aa5cd38b8fad

Reply:
it is a rounding error based on the fact that one api returns unadjustedFee (without multiplier) and the other returns adjusted (with multiplier). In our case we have a constant multiplier of 1, but due to rounding issues we get a +1